.taskcluster.yml

```diff
@@ -72,10 +73,6 @@ tasks:
                 source: ${event.repository.url}
               payload:
                 image:
-                  $if: browser.name == 'webkitgtk_minibrowser'
-                  then:
-                    cl0p3z/web-platform-tests:0.1
-                  else:
                     harjgam/web-platform-tests:0.33
```

Comment:
Indeed, It will be nice if it can be bumped. But I was planning to do that after this lands because I can only have a docker image uploaded and if I upload a new web-platform-tests:0.34 now then the previous based on debian that is running now would be deleted.

Even if the image is not bumped, nightly builds can run ok. The only difference is that they take more time at test time because the dependencies have to be installed at test time for each taskcluster run.

Reply:
I think landing Dockerfile changes with image bumps at the same time makes the most sense, since otherwise there's a time when the Dockerfile in the tree doesn't reflect what's actually being used, and one can't figure out from a version like 0.33 or 0.34 from what it was built.

Reply:
That is a good idea. I just bump the reference and land the PR myself to ensure everything is synced. Thanks for the tip.

Reply:
Bumped in 33240b88f8c5e17f89aedfca34eb2c387d2fb5b3